Enhanced the contact section & added the footer

shiv-chan opened the pull request:
I updated the contact section and also added the footer on the home page.
The footer has the link to our repository.

Thanks.

Dannyboy222 reviewed (approved):
👍

miyabitanimchi commented on client/src/common/components/Footer.tsx:
```diff
@@ -0,0 +1,21 @@
+		<footer className="flex justify-center items-center bg-header_color h-mobileHeaderHeight md:h-laptopHeaderHeight">
+			<p className="text-primary_bg_color text-center text-xs">
+				Copyright ©2021.{' '}
+				<Link
+					to="https://github.com/shiv-chan/senpai-kohai"
```
Maybe we can just use a `a` tag here instead of `Link`? I couldn't find the exact information about this on the docs of React Router, but probably `Link` is not really for external links.
And I clicked the link but now the URL looks like below 😮 `http://localhost:3000/https://github.com/shiv-chan/senpai-kohai`

shiv-chan replied:
I mixed up this with Next.js...😂
Yes, we wanna replace it with `<a>` tag.  Thank you for your review @miyabitanimchi !

miyabitanimchi reviewed (commented):
@shiv-chan  Thank you again for your hard work! The design of the contact section and the footer look great ✨
 I just left one comment below, so please check when you can 👍

miyabitanimchi commented:
Hi @shiv-chan , I changed the `Link` to a `a` tag on `Footer.tsx`. I added `target="_blank"` so we can open a new tab, but was that ok?

Also I didn't notice before but there was the warning below, it was because that earlier `address` tag was inside of `p` tag on the contact section on `Home.tsx` like below.
```
<p className="mb-2">
  Please reach at{' '}
     <address className="font-bold">
         info.senpai.kohai@gmail.com
     </address>
</p>
```
So I moved the `address` outside of the `p`, so please check the `Home.tsx` file before merging and see if the layout looks the same.
By the way I didn't know `address` tag existed. It was good to learn, thank you!🙌

<img width="310" alt="スクリーンショット 2021-12-14 午後2 21 02" src="https://user-images.githubusercontent.com/69213541/146089772-c71f4dfc-0ee3-48ef-8242-3784a624b84f.png">

shiv-chan commented:
@miyabitanimchi 
> I changed the Link to a a tag on Footer.tsx. I added target="_blank" so we can open a new tab, but was that ok?
It looks okay to me 👍 

and thanks for resolving the issue about `address`.  I'm not sure using address tag here is the best way. maybe we could have used just span tag instead.
Anyway, we can leave the code like you fixed for now.

Thanks!
